web/viewer.css

```diff
@@ -1419,6 +1419,7 @@ html[dir='rtl'] .outlineItemToggler::before {
 .dialog .buttonRow {
   text-align: center;
   vertical-align: middle;
+  clear: both;
```

Comment:
It did make a visual difference for the print dialog button, but I didn't spend any time investigating why that is. When I get home tonight I'll take another look!